en/docs/updates/update-tool.md

```diff
@@ -1,17 +1,16 @@
 ## What is the Update Tool?
 
-Update tool is a CLI distribution that is separate for Linux, MacOS and Windows distributions. Bringing your product up to date 
-with the Update Tool is very easy.It is as simple as running one of the commands that matches with your Operating System.
+The Update Tool is a CLI distribution for Linux, MacOS and Windows distributions that streamlines updating WSO2 products. The Update Tool will automatically download the latest updates for your WSO2 products and install them. It will also take a backup of your current installation before updating it.
```

Comment:
We cant say "The Update Tool will automatically download the latest updates", the user has to run it

Reply:
Cool; changed.